plugins/package-managers/go/src/test/kotlin/GoModTest.kt

```diff
@@ -69,4 +71,21 @@ class GoModTest : WordSpec({
             )
         }
     }
+
+    "escapeVersion" should {
+        "escape uppercase letters" {
+            val version = "v0.1.0-M4.0.20231102094829-08e0c3cd016c"
+            val escapedVersion = escapeVersion(version)
```

Comment:
Nit: The empty line before this one is still missing (the code that performs the action to test should stand by its own).

Reply:
Done.